prow/cmd/deck/badge.go

```diff
@@ -85,10 +85,10 @@ func makeShield(subject, status, color string) []byte {
 
 // pickLatestJobs returns the most recent run of each job matching the selector,
 // which is comma-separated list of globs, for example "ci-ti-*,ci-other".
-// jobs will be sorted
+// jobs will be sorted by StartTime in reverse order to display recent jobs first
 func pickLatestJobs(jobs []prowapi.ProwJob, selector string) []prowapi.ProwJob {
 	sort.Slice(jobs, func(i, j int) bool {
-		return jobs[i].Status.StartTime.Before(&jobs[j].Status.StartTime)
+		return jobs[j].Status.StartTime.Before(&jobs[i].Status.StartTime)
```

Comment:
Can you please use `StartTime.After(...` rather than inversing the index usage to make this easier to read?

Also, please update `TestPickLatest` to include timestamps to make sure we don't regress here.
/ok-to-test

Reply:
1. Currently `k8s.io/apimachinery/pkg/apis/meta/v1` Doesn't have `.After()` only `.Before()`. Converting to `time.time` and Back to `meta.Time` might be a little slow, but can be done.

2.  Added tests!